articles/ensemble-bias-var/index.md

```diff
@@ -0,0 +1,125 @@
+ ![hero](/engineering-education/ensemble-bias-var/hero.jpg)
+
+[Source](https://images.unsplash.com/photo-1556884201-c949a3bbf6ad?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=878&q=80)
+
+In my previous [article](/engineering-education/ensemble-learning/), we covered ensemble learning and how it improves machine learning. We mentioned that it reduces the errors of bias and variance. These errors have an impact on the performance of a machine learning model. It is preferable to have low bias and variance. In this article, we dissect the two errors and the effect of ensemble learning on them.
+
+### Contents
+
+1. Overview of bias and variance errors
+2. Relationship between bias and variance
+3. Effect of ensemble methods on bias and variance
+
+### Prerequisites
+
+Aside from the aforementioned [article](/engineering-education/ensemble-learning/) on ensemble methods, I recommend understanding machine learning basics. This [post](/engineering-education/supervised-learning-algorithms/) should help with that.
+
+### Prediction Errors
+
+Prediction errors are defined as the collection of bias, variance, and irreducible errors. We can refer to the irreducible error as noise since we cannot reduce it, regardless of the algorithm chosen. The focus of this article is on the errors of bias and variance. The goal is to understand the two errors. We shall explore their relationship with each other and how ensemble methods affect them.
+
+#### Bias and Variance
+
+**Bias Error**. The bias error of a model is the difference between estimated and actual results. When the bias is high, the model is most likely not learning enough from the training data. The resulting predictions are unreliable, and the generalization is poor. This is what is known as underfitting. To solve this, it is advisable to use a more complex model. However, that is not our focus today. Rather, we shall explore how to deal with bias using boosting.
```

Comment:
What does bias actually mean? When we say bias is high, we are referring to the fact that the model selected was highly biased towards the problem it was trying to solve. The higher the bias, the model has not learned the key features and therefore has resulted in a model that underfits. Makes sense?

Reply:
It does, thank you. Included this in my new definintion